backend/compact-connect/lambdas/purchases/data_model/schema/jurisdiction.py

```diff
@@ -0,0 +1,70 @@
+# pylint: disable=invalid-name
+from marshmallow import pre_dump, Schema
+from marshmallow.fields import String, Decimal, Boolean, Nested, List
+from marshmallow.validate import Length, OneOf
+
+from config import config
+from data_model.schema.base_record import BaseRecordSchema, ForgivingSchema
+
+JURISDICTION_TYPE = 'jurisdiction'
+
+class JurisdictionMilitaryDiscountSchema(Schema):
+    active = Boolean(required=False, allow_none=False)
+    discountType = String(required=False, allow_none=False, validate=OneOf(['FLAT_RATE']))
+    discountAmount = Decimal(required=False, allow_none=False)
+
+class JurisdictionJurisprudenceRequirementsSchema(Schema):
+    required = Boolean(required=False, allow_none=False)
+
+@BaseRecordSchema.register_schema(JURISDICTION_TYPE)
+class JurisdictionRecordSchema(BaseRecordSchema):
+    """
+    Schema for the root jurisdiction configuration records
+    """
+    _record_type = JURISDICTION_TYPE
+
+    # Provided fields
+    jurisdictionName = String(required=True, allow_none=False)
+    postalAbbreviation = String(required=True, allow_none=False, validate=OneOf(config.jurisdictions))
+    compact = String(required=True, allow_none=False, validate=OneOf(config.compacts))
+    jurisdictionFee = Decimal(required=True, allow_none=False)
+    militaryDiscount = Nested(JurisdictionMilitaryDiscountSchema(), required=False, allow_none=False)
```

Comment:
Can we make clear in the docs that this nested config is not required, or otherwise make this required?

Reply:
I've updated the docs to show that this field is optional

Reply:
Thank you!